src/EditorFeatures/CSharpTest/CSharpEditorServicesTest.csproj

```diff
@@ -230,6 +230,7 @@
     <Compile Include="Diagnostics\NamingStyles\NamingStylesTests_OptionSets.cs" />
     <Compile Include="Diagnostics\PreferFrameworkType\PreferFrameworkTypeTests.cs" />
     <Compile Include="Diagnostics\PreferFrameworkType\PreferFrameworkTypeTests_FixAllTests.cs" />
+    <Compile Include="Diagnostics\RemoveUnusedVar\RemoveUnusedVarTest.cs" />
```

Comment:
"Var" implies that this is literally about removing "var".  If this is about removing general variables, then this should be "Variable" and not be abbreviated to "Var".